### DISCUSSION TITLE
Training PINNs on multiple geometries

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [lululxvi/deepxde#856] [Q&A]
So far, from my understanding, `deepxde` is capable of solving a PDE on a "single" geometry. Suppose we want to use the trained PINN on an unseen geometry, what would the training workflow look like?

As an example/use case, consider solving the Poisson equation (with constant source term) on a square with a hole right in the middle with Dirichlet BCs: $u(\Omega u) = 0$, except that the shape of the hole could be a regular n-sided polygon with arbitrary $n$. For this problem, what would the training workflow be like?

I'd really appreciate any comment/suggestion. Thanks!

**Comment by lululxvi** [comment]
You can use DeepONet.
